### PR TITLE
Feat/video player youtube

### PR DESCRIPTION
**Pull request by oegea**
### Description, Motivation and Context

**Context**
Working on a new feature that will involve playing videos from different sources (YouTube, Vimeo, and others...)

**Description**
Add a new component (videoPlayer) that will be in charge of playing videos from different providers. 
Please note that this component IS STILL WIP. We have added basic YouTube support. But we'll keep working during these two weeks.

**Attributions**
Co-author with 💙 @davidmartin84 

### Types of changes

- [X] ✨ New feature (non-breaking change which adds functionality)

### CODE REVIEW

**Review by davidmartin84 — APPROVED** [review]

File: components/atom/videoPlayer/demo/index.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import AtomVideoPlayer from 'components/atom/videoPlayer/src'
+// export default () => <AtomVideoPlayer src='https://www.youtube.com/watch?v=1gI_HGDgG7c' />
```
Comment (andresin87) [inline components/atom/videoPlayer/demo/index.js:2]: why is this line comented?

File: components/atom/videoPlayer/src/index.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  return (
+    <div className="react-AtomVideoPlayer">
+      <Switcher src={src} />
+    </div>
+  )
+}
+
+AtomVideoPlayer.displayName = 'AtomVideoPlayer'
+AtomVideoPlayer.propTypes = {
+  src: PropTypes.string
```
Comment (andresin87) [inline components/atom/videoPlayer/src/index.js:14]: document

File: components/atom/videoPlayer/demo/index.js (hunk 1 above)
Comment (andresin87) [reply components/atom/videoPlayer/demo/index.js:2]: Try to follow the same react template pattern as others demos does (take a look on the padding and title, description, articles, default...)

Comment (oegea) [reply components/atom/videoPlayer/demo/index.js:2]: Thank you for giving feedback.

We take notes and we'll improve it during the following days. Please note that this component is still WIP and we are actively working on it. Current demo doesn't reflect the final one. Other parts of the components will change too during its development.

File: components/atom/videoPlayer/README.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
+# AtomVideoPlayer
+
+> Description
```
Comment (nucliweb) [inline components/atom/videoPlayer/README.md:3]: Description?

Comment (oegea) [reply components/atom/videoPlayer/README.md:3]: Thank you for the feedback.

This is still WIP and we'll take care of it during the following days.

Comment (andresin87) [reply components/atom/videoPlayer/README.md:3]: yeap. its a TODO for those who code it

**Pull request merged**